backend/tests/test_add_item.py

```diff
@@ -89,6 +88,102 @@ def test_add_new_item_with_image(client):
     assert new_price.downvotes == []
 
 
+def test_add_new_item_with_image(client, db):
+    nonexistent_item = test_data.item13
+
+    name = str(nonexistent_item.name)
+    upc = str(nonexistent_item.upc)
+    image = str(base64.b64encode(nonexistent_item.image))
+
+    store = nonexistent_item.stores[0]
+    store_name = str(store.name)
+    lat = float(store.location['lat'])
+    long_arg = float(store.location['long'])
+
+    price_data = store.prices[0]
+    price = float(price_data.price)
+    user = str(price_data.user)
+
+    rv = client.post('/item', data=json.dumps({
```

Comment:
It's unclear why you used rv as your variable name. (Could just be our lack of familiarity with the details of your project)

Reply:
rv is short for return value. It is a common variable name used in Flask. A more descriptive name might be "response", but we use "response" in our test to refer to the JSON data of rv (which is what we actually care about). I was actually not sure of the meaning of rv myself (we just followed examples in Flask documentation), so thanks for teaching me something!

tl;dr: No change, common variable name in Flask